k8s/services/rabbitmq/helm-3.4.cue

```diff
@@ -0,0 +1,294 @@
+		existingWebhookCertSecret:   string | *""
+		existingWebhookCertCABundle: string | *""
+		resources:                   v1.#ResourceRequirements | *{
+			limits: {}
+			requests: {}
+		}
+		podSecurityContext: v1.#PodSecurityContext | *{
+			enabled: true
+			fsGroup: 1001
+		}
+		containerSecurityContext: v1.#PodSecurityContext | *{
+			enabled:                true
+			runAsUser:              1001
+			runAsNonRoot:           true
+			readOnlyRootFilesystem: true
+		}
+		command:          v1.#Command | *[]
+		args:             v1.#Args | *[]
+		fullnameOverride: string | *""
+		hostAliases: []
+		podLabels:             k8s.#Labels
+		podAnnotations:        k8s.#Annotations
+		podAffinityPreset:     string | *""
+		podAntiAffinityPreset: string | *"soft"
+		nodeAffinityPreset:    v1.#Affinity
+		affinity:              v1.#Affinity
+		nodeSelector:          v1.#NodeSelector
+		tolerations: [...v1.#Toleration]
+		updateStrategy: {
+			type: string | *"RollingUpdate"
+		}
+		priorityClassName: string | *""
+		lifecycleHooks: {}
+		containerPorts: {
+			metrics: k8s.#Port | *8080
+		}
+		extraEnvVars: []
+		extraEnvVarsCM:     string | *""
+		extraEnvVarsSecret: string | *""
+		extraVolumes: []
+		extraVolumeMounts: []
+		sidecars: []
+		initContainers: []
+		rbac: {
+			create: bool | *true
+		}
+		serviceAccount: #ServiceAccount
+		metrics:        #Metrics
+
+		service: k8s.Service | *{
+			type:  string | *"ClusterIP"
+			ports: k8s.#ServicePort | *{
+				webhook: 443
+			}
+			nodePorts: k8s.#ServicePort | *{
+				http: ""
+			}
+			clusterIP:      string | *""
+			loadBalancerIP: string | *""
+			extraPorts: []
+			loadBalancerSourceRanges: []
+			externalTrafficPolicy: string | *"Cluster"
+			annotations: {}
+			sessionAffinity: string | *"None"
+			sessionAffinityConfig: {}
+		}
+	}
+	useCertManager: bool | *false
+}
+
+#ServiceMonitor: {
+	enabled:     bool | *false
+	jobLabel:    k8s.#Labels | *"app.kubernetes.io/name"
+	honorLabels: false
+	selector: {}
+	scrapeTimeout: string | *""
+	interval:      string | *""
+	metricRelabelings: []
+	relabelings: []
+	labels: {}
+}
+
+#ServiceAccount: {
+	// -- Specifies whether a service account should be created.
+	create: bool | *true
+	// -- Annotations to add to the service account.
+	annotations: k8s.#Annotations
+	// -- Extra Labels to add to the service account.
+	extraLabels: k8s.#Labels
+	// -- The name of the service account to use.
+	// If not set and create is true, a name is generated using the fullname template.
+	name:                         string | *""
+	automountServiceAccountToken: bool | *true
+}
+
+#Metrics: service: {
+	// -- Enable if you use another monitoring tool than Prometheus to scrape the metrics
+	enabled: bool | *false
+
+	type: string | *"ClusterIP"
+
+	// -- Metrics service port to scrape
+	port:      k8s.#Port | *80
+	nodePorts: k8s.#Port | *{
+		http: ""
+	}
+	clusterIP: string | *""
+	extraPorts: []
+	loadBalancerIP: string | *""
+	loadBalancerSourceRanges: []
```

Comment:
what's the schema? maybe [...string]

if it's not know we can leave it as anything [..._]